packages/as-types/assembly/argument.ts

```diff
@@ -325,9 +454,12 @@ export class Args {
   }
 
   /**
-   * Returns the data of requested size for current offset
+   * This function retrieves the next chunk of data from the serialized data array based on the specified size
+   * and returns it as a static array of unsigned 8-bit integers.
```

Comment:
Maybe add a @remarks to explain that calling this function (unlike other 'next') doesn't increment the offset and therefore is quite usefull to fetch data directly and still be able to use other functions afterwards.
And also that using this function might produce duplicates of the data fetched for the same reasons.